.azuredevops/modulePipelines/ms.appconfiguration.configurationstores.yml

```diff
@@ -0,0 +1,55 @@
+name: 'AppConfiguration: ConfigurationStores'
```

Comment:
```suggestion
name: 'App Configuration - ConfigurationStores'
```
Otherwise not supported

Reply:
True for the` :` part. However for consistency I'd suggest without a space:
```suggestion
name: 'AppConfiguration - ConfigurationStores'
```

Reply:
Updated to `name: 'AppConfiguration - ConfigurationStores'`